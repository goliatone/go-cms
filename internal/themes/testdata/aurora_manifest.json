{
  "name": "aurora",
  "description": "Default marketing theme with hero-focused layout",
  "version": "1.0.0",
  "author": "Go CMS",
  "widget_areas": [
    {
      "code": "header.global",
      "name": "Global Header",
      "scope": "global"
    },
    {
      "code": "footer.global",
      "name": "Global Footer",
      "scope": "global"
    }
  ],
  "menu_locations": [
    {
      "code": "primary_navigation",
      "name": "Primary Navigation"
    },
    {
      "code": "footer_links",
      "name": "Footer Links"
    }
  ],
  "assets": {
    "base_path": "public",
    "styles": [
      "css/base.css",
      "css/aurora.css"
    ],
    "scripts": [
      "js/site.js"
    ],
    "images": [
      "images/logo.png"
    ]
  },
  "metadata": {
    "preview_image": "images/previews/landing.png",
    "support_email": "support@example.com"
  }
}

