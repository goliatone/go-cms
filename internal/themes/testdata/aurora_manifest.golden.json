{
  "Name": "aurora",
  "Description": "Default marketing theme with hero-focused layout",
  "Version": "1.0.0",
  "Author": "Go CMS",
  "ThemePath": "themes/aurora",
  "Config": {
    "widget_areas": [
      {
        "code": "header.global",
        "name": "Global Header",
        "scope": "global"
      },
      {
        "code": "footer.global",
        "name": "Global Footer",
        "scope": "global"
      }
    ],
    "menu_locations": [
      {
        "code": "primary_navigation",
        "name": "Primary Navigation"
      },
      {
        "code": "footer_links",
        "name": "Footer Links"
      }
    ],
    "assets": {
      "base_path": "public",
      "styles": [
        "css/base.css",
        "css/aurora.css"
      ],
      "scripts": [
        "js/site.js"
      ],
      "images": [
        "images/logo.png"
      ]
    },
    "metadata": {
      "preview_image": "images/previews/landing.png",
      "support_email": "support@example.com"
    }
  },
  "Activate": false
}
